src/genophenocorr/model/_variant.py

```diff
@@ -423,6 +423,20 @@ def tx_annotations(self) -> typing.Sequence[TranscriptAnnotation]:
     @property
     def genotypes(self) -> Genotypes:
         return self._gts
+    
+    def get_hgvs_cdna_by_tx(self, transcript_id:str) -> str:
```

Comment:
Please verify if the typing annotation for the return type (and the documentation) is correct.

Reply:
updated documentation and set return type to Optional[str]